src/Goerli-DssSpell.t.sol

```diff
@@ -2556,34 +2557,34 @@ contract DssSpellTest is DSTest, DSMath {
         // Insert ilk offboarding tests here
```

Comment:
I think we can comment the following unused tests to reduce testing time.
- `testCollateralIntegrations`
- `testNewChainlogValues`
- `testNewIlkRegistryValues`
- `testIlkOffboarding`